evansmusomi/react-native-102
# Styles Nav and Makes Place Detail Responsive

*What this does*
- Styles navigation elements
- Adds responsiveness to Place Detail screen

*How to verify*
- The navigation tabs and side drawer icon on the Find and Share place screens should be orange
- The layout of the Place Detail screen changes you switch between portrait and landscape view
